### ISSUE TITLE
Translate `reference/simple_stmts.po`

### ISSUE THREAD

**Issue by cmaureir** [python/python-docs-es#2643]
This needs to reach 100% translated.

The rendered version of this file will be available at https://docs.python.org/es/3.12/reference/simple_stmts.html once translated.
Meanwhile, the English version is shown.

Current stats for `reference/simple_stmts.po`:

* Fuzzy: 16
* Percent translated: 60.3%
* Entries: 167 / 277
* Untranslated: 110

Please, comment here if you want this file to be assigned to you and an member will assign it to you as soon as possible, so you can start working on it.

Remember to follow the steps in our [Contributing Guide](https://python-docs-es.readthedocs.io/page/CONTRIBUTING.html).

**Comment by psolenny** [comment]
Hola!, me gustaría contribuir trabajando en este archivo. 🙋‍♀️

**Comment by rtobar** [comment]
Muchas gracias por el entusiasmo @psolenny, y por las ganas de comenzar a contribuir al proyecto!

Te había asignado ya este archivo, pero luego me fijé que es uno bastante grande, con más de 100 entradas por traducir. Si bien no es imposible, dado que sería tu primera contribución yo te recomendaría ir por uno más simple, y así puedes ir agarrando un poco de experiencia no sólo con la traducción en si, sino que con todo el proceso de revisión y testeo. Hay muchos issues para elegir, con la gran mayoría etiquetados como "good first issue", así que hay opciones para regodearse.

**Comment by jcaalzago** [comment]
Buenas, 

me gustaría poder traducir este archivo. No me importa el número de entradas por traducir.

**Comment by cmaureir** [comment]
Hola @jcaalzago mejor dejamos que @psolenny lo termine

**Comment by jcaalzago** [comment]
Claro, sin problema, al no ver a nadie asignado, pensé que estaba libre :)
